src/drgn/src/drgn/kafka.py

```diff
@@ -38,27 +38,38 @@ def _get_topic_partition(self, topic: str):
         return TopicPartition(topic, 0, 0)
 
     def consume(self, topic: str) -> Generator[list[dict], None, None]:
-        self._consumer_config["auto.offset.reset"] = env_config["consumer"][
-            "offset_reset"
-        ]
+        self._consumer_config["auto.offset.reset"] = "earliest"
+        self._consumer_config["enable.auto.commit"] = False
         self.consumer.assign([self._get_topic_partition(topic)])
-        consume_size = int(env_config["consumer"]["consume_size"])
-        consume_timeout = float(env_config["consumer"]["consume_timeout"])
+        if isinstance(env_config, dict):
```

Comment:
No way you add that here, why would they not be an env_config ? The whole repo is setup to have one